wordpress-mobile/WordPress-iOS
# Remove AFNetworking from project.

This PR removes the last remaining references and uses of AFNetworking from the project.

The uses were mainly the use of AFNetworking extension for UIImageView that allows to set URLs and download images on the background for the view.

I replaced these calls by using a similar extension that was already implemented on WordPressUI on a previous sweep to remove AFNetworking.

One thing that I remove was a call on the App Delegate that was making AFNetworking manage the network indicator. I could replace it by using [AlamoFire network indicator sub pod](https://github.com/Alamofire/AlamofireNetworkActivityIndicator) but do we want to do this? Keep in mind that the majority calls where now being done using AlamoFire so this was not being almost used. What do you say?

To test:
 - Check the reader section of the app.
 - Check the post and blog/sites  list section, specially the site and user icons in there.

Update release notes:
